### PR TITLE
Added changes for Azure MAS + CP4D deployment

### PR DESCRIPTION
**Pull request by vaibhavkulkarniibm**

### CODE REVIEW

File: azure/azurefiles-premium.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+driver_version=v1.12.0
+echo "Driver version " $driver_version
+curl -skSL https://raw.githubusercontent.com/kubernetes-sigs/azurefile-csi-driver/$driver_version/deploy/install-driver.sh | bash -s $driver_version --
+
+#Deploy premium Storage Class
+wget -nv https://raw.githubusercontent.com/Azure/maximo/main/src/storageclasses/azurefiles-premium.yaml -O /tmp/OCPInstall/azurefiles-premium.yaml
+envsubst < /tmp/OCPInstall/azurefiles-premium.yaml > /tmp/OCPInstall/QuickCluster/azurefiles-premium.yaml
+oc apply -f /tmp/OCPInstall/QuickCluster/azurefiles-premium.yaml
+
+oc apply -f https://raw.githubusercontent.com/Azure/maximo/main/src/storageclasses/persistent-volume-binder.yaml
```
Comment (santoshjpawar) [inline azure/azurefiles-premium.sh:35]: We should keep the files in our repo. We cannot refer to MS team's repo during the production deployments.

Comment (vaibhavkulkarniibm) [reply azure/azurefiles-premium.sh:35]: Hi @santoshjpawar 
Is it like, we have to keep these 3 files checked-in inside Azure directory of our repo for when `ENV_TYPE="prod"` 
and use URL's when `ENV_TYPE="dev"` 
https://raw.githubusercontent.com/Azure/maximo/main/src/storageclasses/azure.json
https://raw.githubusercontent.com/Azure/maximo/main/src/storageclasses/azurefiles-premium.yaml
https://raw.githubusercontent.com/Azure/maximo/main/src/storageclasses/persistent-volume-binder.yaml

Or, to have it checked-in repo for both prod and dev case ?

Comment (santoshjpawar) [reply azure/azurefiles-premium.sh:35]: Hi @vaibhavkulkarniibm  We will always keep the files in our repo for all types of deployments.

Comment (santoshjpawar) [reply azure/azurefiles-premium.sh]: Looks good.

**Pull request merged**